doc/adr/0007-use-swiftsoup-to-parse-open-graph-metadata.md

```diff
@@ -0,0 +1,86 @@
+# 7. Use SwiftSoup to parse Open Graph metadata
+
+Date: 2024-10-02
+
+Authors: Josh Brown
+
+## Status
+
+Accepted
+
+## Context & Problem Statement
+
+In order to display a video in portrait or landscape orientation based on its [Open Graph](https://ogp.me/) metadata, we need to parse the HTML document containing the video. The Open Graph metadata contains properties that specify the width and height of the video, and similar properties for an image that may be used as a cover image
```

Comment:
Looks like a fullstops is missing at the end of this sentence.

Reply:
Good eye! Thanks for catching this!

Reply:
Fixed!